Makefile

```diff
@@ -55,7 +55,11 @@ else
 	CFLAGS += $(shell $(PKG_CONFIG) --cflags valgrind)
```

Comment:
Patch looks fine, but you're missing a signed-off-by line.

Reply:
Added a Sob-lines for all commits now. I suggest installing https://github.com/apps/dco for your repo, this adds an automatic check for missing Sob-lines.